docs/index.rst

```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
 This installs the executables ``pathfinding-service``, ``monitoring-service``
 and ``request-collector`` into the virtual environment.
 
+Using MS and PFS in raiden
+--------------------------
+We need to install raiden from the latest development branch.
+
+.. code:: bash
+
+    git clone https://github.com/raiden-network/raiden.git
+    cd raiden
+    git checkout development
+    make install-dev
+
+Raiden client needs to be started with the raiden-services flags like
+
+* --enable-monitoring
+* --routing-mode [pfs|local|private]
+
+Doing ``raiden --help`` and going to Raiden Services options would give more
+information.
+
+Payments related to Services
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The monitoring service registration, client onboarding(raiden node asking for
+a raiden service), monitoring service payment requires acquiring tokens in the
+token contracts deployed in various testnets and the mainnet. All deployment
+data regarding the various raiden-service contracts and associated tokens is
+displayed in the prefixed ``data`` folders here_. One can read about the design
+of ``raiden-services`` in the specs_.
+
+.. _here: https://github.com/raiden-network/raiden-contracts
+    /tree/master/raiden_contracts
+.. _specs: https://raiden-network-specification.readthedocs.io/en/latest
+    /monitoring_service.html#design-of-the-monitoring-service-still-work-in-progress
+
+
+
+For example the latest deployment of the raiden-service contracts_
+on the goerli testnet.
+
+.. _contracts: https://github.com/raiden-network/raiden-contracts
+  /blob/master/raiden_contracts/data_0.17.0/deployment_services_goerli.json
+
+
+The various *raiden-service* contracts and associated tokens can be deciphered
+from the json file containing the deployed data. Thus navigating to the
+``data_0.17.0`` folder and opening the ``deployment_services_goerli.json``
+shows us the following information.
+
++------------------+----------------------------------------------------------+
```

Comment:
Not sure if this really helpful, but if we leave it I'd recommend to make all addresses links to etherscan.

Reply:
yes I will include the links to etherscan thanks for reminding